### PR TITLE
Use kubuconfig arg with prowgen

### PR DESCRIPTION
**Pull request by hongkailiu**
Based on https://github.com/openshift/release/pull/5934#issuecomment-554383694

/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

**Comment by hongkailiu** [comment]
/hold

Need to prepare the generated jobs.

File: cmd/ci-operator-prowgen/main.go [hunk 1]
```diff
@@ -31,6 +31,13 @@ const (
 	sentryDsnMountPath  = "/etc/sentry-dsn"
 	sentryDsnSecretPath = "/etc/sentry-dsn/ci-operator"
 
+	apiciCIOperatorMountName       = "apici-ci-operator-credentials"
```
Comment (stevekuznetsov) [inline cmd/ci-operator-prowgen/main.go]: If these are used once, we should just inline the literal string.

Comment (hongkailiu) [reply cmd/ci-operator-prowgen/main.go]: Done.

**Comment by hongkailiu** [comment]
> Can we hard-code this generation update for one repo, first?

On it.

**Comment by hongkailiu** [comment]
> > Can we hard-code this generation update for one repo, first?
> 
> On it.

https://github.com/openshift/release/pull/5945

**Comment by hongkailiu** [comment]
will cancel on Monday. https://github.com/openshift/release/pull/5945/files is the update for jobs.

**Pull request merged**